compiler/executable/match_/planner/plan.rs

```diff
@@ -1530,7 +1530,7 @@ impl ConjunctionPlan<'_> {
 
             PlannerVertex::Negation(negation) => {
                 let negation = negation.plan().lower(
-                    match_builder.current_outputs.iter().copied(),
+                    match_builder.row_variables().iter().copied(),
```

Comment:
So this should simply have been `produced_so_far` as in the other lowering steps?

Reply:
Covered in person; `produced_so_far` captures all produced variables by this point in the plan, including ones that were discarded.